examples/tutorials/region-arguments.py

```diff
@@ -0,0 +1,156 @@
+    borders="1/0.5p",
+    shorelines="1/0.5p",
+    frame="ag",
+)
+fig.show()
+
+########################################################################################
+#
+# GMT has the option for region coordinates to be passed in classic mode, which was the
+# standard prior to GMT version 6. The string format takes the coordinates for the
+# bottom-left and top-right coordinates. To specify classic mode, append **+r** at the
+# end of the ``region`` string.
+
+fig = pygmt.Figure()
+fig.coast(
+    # Sets the bottom-left corner as 10E, 35N and the top-right corner as 20E, 45N
+    region="10/35/20/45+r",
+    projection="M12c",
+    land="lightgray",
+    water="white",
+    borders="1/0.5p",
+    shorelines="1/0.5p",
+    frame="ag",
+)
+fig.show()
+
+########################################################################################
+# Global regions
+# --------------
+#
+# In addition to passing coordinates, the argument **d** can be passed to set the
+# region to the entire globe. The range is 180W to 180E (-180, 180) and 90S to
+# 90N (-90 to 90). With no parameters set for the projection, the figure defaults to be
+# centered at the mid-point of both x- and y-axes. Using **d**\ , the figure is
+# centered at 0,0, or the intersection of the equator and prime meridian.
+
+fig = pygmt.Figure()
+fig.coast(
+    region="d",
+    projection="Cyl_stere/12c",
+    land="lightgray",
+    water="white",
+    borders="1/0.5p",
+    shorelines="1/0.5p",
+    frame="ag",
+)
+fig.show()
+
+########################################################################################
+#
+# The argument **g** can be passed, which encompasses the entire globe. The range is
+# 0E to 360E (0, 360) and 90S to 90N (-90 to 90). With no parameters set for the
+# projection, the figure is centered at 180,0, or the intersection of the equator and
+# International Date Line.
+
+fig = pygmt.Figure()
+fig.coast(
+    region="g",
+    projection="Cyl_stere/12c",
+    land="lightgray",
+    water="white",
+    borders="1/0.5p",
+    shorelines="1/0.5p",
+    frame="ag",
+)
+fig.show()
+
+########################################################################################
+# ISO code
+# --------
+#
+# The ``region`` can be set to include a specific area specified by the two-character
+# `ISO 3166-1 alpha-2 convention
+# (for futher information: https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2).
+
+fig = pygmt.Figure()
+fig.coast(
+    # Sets the figure region to encompass Japan with the ISO code "JP"
+    region="JP",
+    projection="M12c",
+    land="lightgray",
+    water="white",
+    borders="1/0.5p",
+    shorelines="1/0.5p",
+    frame="ag",
+)
+fig.show()
+
+########################################################################################
+#
+# The area encompassed by the ISO code can be expanded by appending **+r**\ *increment*
+# to the ISO code. The *increment* unit is in degrees and expands the range of the
+# region in all directions.
```

Comment:
**+e** and **+R** also allowed, and *increment* can be different for 4 directions.

https://docs.generic-mapping-tools.org/latest/cookbook/options.html#data-domain-or-map-region-the-r-option

Reply:
Added